### PR TITLE
[fix bug 1140335] Filter events by initiative.

### PR DESCRIPTION
**Pull request by akatsoulas**
* Export initiatives in API v1.

### CODE REVIEW

File: remo/events/helpers.py [hunk 1]
```diff
@@ -98,11 +98,13 @@ def get_event_comment_delete_url(obj):
...
-def get_event_category_link(category):
-    """Returns events list page of given category."""
+def get_event_filtered_url(category=None, initiative=None):
+    """Returns events list page of given category or initiative."""
 
     url = reverse('events_list_events')
-    return urlparams(url, '/category/%s/' % category.lower())
+    if category:
+        return urlparams(url, '/category/%s/' % category.lower())
+    return urlparams(url, '/initiative/%s/' % initiative.lower())
```
Comment (johngian) [inline remo/events/helpers.py]: This returns ``'NoneType' object has no attribute 'lower'`` when ``category=None`` and ``initiative=None``.

File: remo/events/views.py [hunk 2]
```diff
@@ -37,10 +38,11 @@ def redirect_list_events(request):
 @cache_control(private=True)
 def list_events(request):
     """List events view."""
-    events = Event.objects.all()
-    categories = FunctionalArea.objects.all()
-    return render(request, 'list_events.html',
-                  {'events': events, 'categories': categories})
+    data = {'events': Event.objects.all(),
+            'categories': FunctionalArea.objects.all(),
+            'initiatives': Campaign.objects.all()}
```
Comment (johngian) [inline remo/events/views.py]: API returns only active ``campaigns``. We might need to consider this as well here.

**Comment by johngian** [comment]
It looks like there is a circular dependency. I am getting this error locally:
``cannot import name EventResource``

**Comment by akatsoulas** [comment]
Probably, this is happening because you have some leftovers folders with some *.pyc files inside from the remo-api branch.

**Comment by johngian** [comment]
lgtm r+wc

**Pull request merged**